pinot-common/src/main/java/org/apache/pinot/common/metrics/MetricsHelper.java

```diff
@@ -48,7 +48,7 @@
       new WeakHashMap<MetricsRegistryRegistrationListener, Object>();
 
   /**
-   * Initializes the metrics system by initializing the registry registration listeners present in the configuration.
+   * Initializes the metrics system by initializing the registry rePinotJmxMetricsReporter is startedgistration listeners present in the configuration.
```

Comment:
??

Reply:
probably accidental copy-paste, will fix this